com.github.taburineagle.NeewerLite-Python.yaml

```diff
@@ -0,0 +1,124 @@
+id: com.github.taburineagle.NeewerLite-Python
+branch: stable
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm13
```

Comment:
Does this not work with a newer llvm version?

Reply:
I try with lv14 it work i push the changes

Reply:
done